Libplanet.Net/Consensus/Context.Mutate.cs

```diff
@@ -286,6 +343,11 @@ private void ProcessGenericUponRules()
                     PublishMessage(
                         new ConsensusPreCommitMsg(
                             MakeVote(Round, p3.Block.Hash, VoteFlag.PreCommit)));
+
+                    // Maybe need to broadcast periodically?
+                    PublishMessage(
+                        new ConsensusMaj23Msg(
+                            MakeMaj23(Round, p3.Block.Hash, VoteFlag.PreVote)));
```

Comment:
Do we have to publish `ConesnsusMaj23Msg` only on condition that we have proper proposal of +2/3 votes?
Just for asking, since other implementation doesn't seem to check proposal condition.
I'm not sure which one would be better, and think this implementation is also good, since it can reduce a number of messages.

Not sure whether `Maj23` should be the flag of `Single correct process`, or `Single +2/3 votes`.